### ISSUE TITLE
Is it possible to configure Self-signed Edge Certs for transparent gateway scenario

### ISSUE THREAD

**Issue by hubez** [Azure/iotedgehubdev#172]
How (or is it possible) to set your own Edge Certs in iotdegehubdev for a gateway scenario? I assume it supports use of leaf devices as a transparent gateway from reading https://github.com/Azure/iotedgehubdev/issues/143

An example leaf device would be the the SimulatedDevice.py sample: https://github.com/Azure-Samples/azure-iot-samples-python/blob/master/iot-hub/Quickstarts/simulated-device/SimulatedDevice.py but configured with a Gateway.

Otherwise, the edgeHubDev keeps giving this error whenever a leaf device connects (presumably because my leaf device is using a cert that doesn't align to edgeHubDev's trust chain):
`
edgeHubDev                     | 2019-02-19 11:57:57.015 +00:00 [WRN] - TLS handshake failed., System.IO.IOException: Channel is closed, 26156441
`

**Comment by hubez** [comment]
Found my answer, I think. Would be useful to have a confirmation from someone in the know.

I found this file in iotedgehubdev that looks to be the cert generation for it: https://github.com/Azure/iotedgehubdev/blob/master/iotedgehubdev/edgecert.py where it references the Root CA location.

In my case on windows, if I have my leaf device use the file `C:\ProgramData\iotedgehubdev\data\certs\edge-device-ca\cert\edge-device-ca-root-cert.pem` it seems to successfully connect.
